packages/components/src/ellipsis-menu/README.md

```diff
@@ -0,0 +1,97 @@
+EllipsisMenu
```

Comment:
The example ellipsis is cut off in Chrome
<img width="500" alt="Screen Shot 2019-09-06 at 11 13 22 AM" src="https://user-images.githubusercontent.com/343847/64434662-70a4d300-d097-11e9-8337-b0a39cca9b63.png">

Reply:
I believe this is an issue for all `EllipsisMenu`s on `master` as well.